pkg/csi/cinder/controllerserver.go

```diff
@@ -266,13 +267,18 @@ func (cs *controllerServer) CreateSnapshot(ctx context.Context, req *csi.CreateS
 		klog.V(3).Infof("CreateSnapshot %s on %s", name, volumeId)
 	}
 
+	ctime, err := ptypes.TimestampProto(snap.CreatedAt)
+	if err != nil {
+		klog.Errorf("Error to convert time to timestamp: %v", err)
```

Comment:
this will return ctime as 0 ? we will use this value even error occur here..

Reply:
yes, it is mandatory to return as type Timestamp as per CSI compliance